drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/PaymentMethodsListViewModel.kt

```diff
@@ -32,105 +30,132 @@ import com.adyen.checkout.dropin.ui.paymentmethods.PaymentMethodListItem
 import com.adyen.checkout.dropin.ui.paymentmethods.PaymentMethodModel
 import com.adyen.checkout.dropin.ui.paymentmethods.PaymentMethodNote
 import com.adyen.checkout.dropin.ui.paymentmethods.StoredPaymentMethodModel
-import com.adyen.checkout.dropin.ui.stored.makeStoredModel
+import com.adyen.checkout.dropin.ui.stored.isStoredPaymentSupported
+import com.adyen.checkout.dropin.ui.stored.mapStoredModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
 
 @Suppress("TooManyFunctions")
-class PaymentMethodsListViewModel(
-    application: Application,
+internal class PaymentMethodsListViewModel constructor(
```

Comment:
`constructor` is not explicitly needed right?

Reply:
no it's not